Bio/SeqIO/InsdcIO.py

```diff
@@ -647,17 +659,20 @@ def _write_the_first_line(self, record):
             # just the generic Alphabet (default for fasta files)
             raise ValueError("Need a DNA, RNA or Protein alphabet")
 
+        topology = self._get_topology(record)
+
         division = self._get_data_division(record)
 
         assert len(units) == 2
         assert len(division) == 3
         # TODO - date
         # TODO - mol_type
-        line = "LOCUS       %s %s %s    %s           %s %s\n" \
+        line = "LOCUS       %s %s %s    %s  %s %s %s\n" \
```

Comment:
Why the double space before the topology? Which columns does the spec say are allocated here?